### DISCUSSION TITLE
Provide a TerraForm template for hosting on cloud providers

### DISCUSSION THREAD

**Discussion by srameshr** [plausible/analytics#710] [Anything Else]
This is more like a feature request for self-hosting. 

It would be cool if there was a Terraform template under the hosting repo for one-click scalable deployment of all the services used by Plausible.

**Comment by ukutaht** [comment]
Would be cool!

> one-click scalable deployment

By this I'm assuming you mean a clustered deployment? At the moment we don't do that even for our production servers. The app server is stateful and no work has been done to support a multi-node deployment. We're scaling vertically for now.

**Comment by srameshr** [comment]
Yes, a clustered deployment.
